modules/register.go

```diff
@@ -23,8 +23,8 @@ func Register(cfg api.Config, manifest *Manifest) (*types.Module, error) {
 	}
 
 	client := api.Client{Config: cfg}
-	if err := client.Org(module.OrgName).Modules().Create(module); err != nil {
```

Comment:
Was this just broken the whole time?

Reply:
The older API used the module.OrgName to determine the API endpoint URL *and* access token retrieval.
This change relies on the API client for access token retrieval and module.OrgName for API endpoint URL.